tests/unit/test_data_utils.py

```diff
@@ -0,0 +1,141 @@
+import pandas as pd
+import pytest
+
+from pseudopeople.constants import paths
+from pseudopeople.constants.metadata import DatasetNames
+from pseudopeople.loader import (
+    combine_joint_filers,
+    flatten_data,
+    load_and_prep_1040_data,
+)
+from pseudopeople.schema_entities import COLUMNS, DATASETS
+
+
+@pytest.fixture(scope="module")
```

Comment:
I tried to keep these dataframes small so it is easier to conceptualize what is actually happening.